### DISCUSSION TITLE
generate emoji

### DISCUSSION THREAD

**Discussion by luisamanett** [lllyasviel/Fooocus#3635] [Q&A] [answered]
Hi all, can anyone suggest prompts to use to generate emoji style? for example an emoji style chick waving, or holding a heart in his hands, in the same position but in 4 different positions with different movements to then create the gif, and what style to use? Thank you.

like this https://cdn.vectorstock.com/i/500p/32/35/funny-yellow-alien-character-emoticons-set-vector-8623235.jpg

**Comment by Yogesh-DevHub** [comment, marked as answer]
Get a emoji LORA like this, https://civitai.com/models/250567/emojis

**Reply by luisamanett** [reply]
many thanks , can you give me a suggestion for a prompt? and the setting? I'm not very good

**Reply by Yogesh-DevHub** [reply]
For this specific case, I suggest go to civit AI, the link I shared above. You will find images of emoji, at the bottom right on the image there is a letter "i" inside a circle.
If you click, you will find the prompt used to generate that specific image, like this. "Owl emoji on a white background" 

Now you can use same or make changes to it. Or give it to GPT and ask him to generate 10 - 15 similar creative prompt for your emoji collection. 
About style and settings, you have to experiment with a few models and lora for your requirement.

I have a playlist on YT: https://youtube.com/playlist?list=PLPFN04WspxqsslRSpiLmwGR8QTpDYNv7z&si=1E1UChFydAQn6m6X 
explaining the prompt basic for fooocus, and how to use styles and settings. The videos are short between 5 - 10 min. It should answer most of your questions. 

This is one more YT channel having good videos for fooocus : https://www.youtube.com/@KLEEBZTECH

**Comment by luisamanett** [comment]
many thanks
